errors.ts

```diff
@@ -99,6 +83,26 @@ export function installUncaughtExceptionListener(actionOnException?: () => void)
 	});
 }
 
+function tryTrackException(error: Error, injector: IInjector): void {
+	let disableAnalytics: boolean;
+	try {
```

Comment:
I do not want to set the global value, the idea is to have it set to `true` just here for skipping AnalyticsService. This is usefult only in our unit tests and we should never set it to true in a real use-case.